packages/cli/src/lib/redis/api.ts

```diff
@@ -66,7 +66,7 @@ export type RedisFormationWaitResponse = {
 type HttpVerb = 'GET' | 'POST' | 'PATCH' | 'DELETE' | 'PUT'
 
 export default (app: string, database: string | undefined, json: boolean, heroku: APIClient) => {
-  const HOST = process.env.HEROKU_REDIS_HOST || 'api.data.heroku.com'
```

Comment:
What do you think of doing something like:

```javascript
const HOST = process.env.HEROKU_DATA_HOST || process.env.HEROKU_REDIS_HOST || 'api.data.heroku.com'
```

for now, and then plan to remove the `process.env.HEROKU_REDIS_HOST` bit in a future major release? I think the concern @k80bowman was alluding to is that possibly someone is relying on that config var being set in a script. The risk is low, I think, but we tend to bundle the removal of things into major releases.

Reply:
agree, the risk is likely scoped to internal staff. we'll create an WI to remove `process.env.HEROKU_REDIS_HOST` when we do the next major version bump, likely to be within a few months from now.